R/page_size.R

```diff
@@ -86,6 +87,7 @@ page_dim <- function(page_type) {
 #' @return named list with `cpi` and `lpi`, the characters and lines per
 #' inch, respectively.
 #'
+#' @keyword internal
```

Comment:
used in `rtables::export_as_pdf` and `rtables::export_as_txt`, must be exported until (at least) export logic is fully migrated to formatters